prices/src/lib.rs

```diff
@@ -0,0 +1,42 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use num_traits::Zero;
```

Comment:
```suggestion
use sr_primitives::traits::Zero;
```

To avoid introducing `num_traits` in cargo toml.